app/views/businesses/_form.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= form_with model: business, url: category_businesses_path, :html => {:class => 'form', :id => 'form' }  do |form| %>
+
+      <div class="field">
+         <%= form.select :category_id, categories.collect { |category| [category.name, category.id]} %><br />
+      <div>
```

Comment:
- [ ] The select input does not select the correct category. I created a transaction and I chose it to be added to a `Tech` category but it was added to the `Shopping` category.